avocado_vt/loader.py

```diff
@@ -101,85 +92,48 @@ class VirtTestLoader(loader.TestLoader):
 
     name = 'vt'
 
-    def __init__(self, args, extra_params):
+    def __init__(self, config, extra_params):
         """
         Following extra_params are supported:
          * avocado_vt_extra_params: Will override the "vt_extra_params"
-           of this plugins "self.args" (extends the --vt-extra-params)
+           of this plugins "self.config" (extends the --vt-extra-params)
         """
```

Comment:
Btw, I didn't have the chance to say that this is one great change. We had to do a lot of inversions of this effect in VT since we wanted the update from Avocado yet dependended on VT not being updated.